wandb/sdk/wandb_run.py

```diff
@@ -585,6 +585,9 @@ def __setstate__(self, state: Any) -> None:
         if not _attach_id:
             return
 
+        run = wandb._attach(_attach_id)
+        for k, v in run.__dict__.items():
+            self.__dict__[k] = v
         self._attach_id = _attach_id
```

Comment:
should we populate both `run_id` and `attach_id` used later to see if the run is attached?
or should we have a new field?